### ISSUE TITLE
Debugging web tests is painful

### ISSUE THREAD

**Issue by rchiodo** [microsoft/vscode-jupyter#9807]
Console logger doesn't work - probably because it isn't enabled. Might need to do the same trick done with the URI for the jupyter server - write the isTestExecution() result into the bundled js file.

Debugging doesn't work because creating files using the 'vscode-test-web' scheme doesn't work. Could probably retry with the 'file' scheme if the first scheme fails.

**Comment by DonJayamanne** [comment]
> Debugging doesn't 

Yes, source maps don't work. This is something i found last time i enabled bundling web tests into the extension. Sux.

**Comment by DonJayamanne** [comment]
> do the same trick done with the URI for the jupyter server 

whats that?

**Comment by DonJayamanne** [comment]
@rchiodo is this still an issue.
I think the PR i recently added resolves this issue, as we can now fully debug Web tests from within VS Code.

**Comment by rchiodo** [comment]
Locally, but not when they fail on CI. I still think we need to log output somewhere (not necessarily to the console).

**Comment by DonJayamanne** [comment]
> but not when they fail on CI. I still think we need to log output somewhere (not necessarily to the console).

I think during debt week can do this easily.
Just re-direct all of the console output to a log file.

Note: This PR https://github.com/microsoft/vscode-jupyter/pull/10512 now logs everything to the console window.

**Comment by DonJayamanne** [comment]
@rchiodo I'm sorry but I do'nt quite understand this statement

> Locally, but not when they fail on CI. I still think we need to log output somewhere (not necessarily to the console).

Currently on CI we have enabled verbose logging, hence all of the logging is in the console window on CI, hence you can download the logs & look at what went wrong. We only log to the console based on some CI env variables.

* So we can get the logs today from the CI job (you can download the output of the job)
* Logging to console is only conditional

What more do we need or not need here.
I have a simple PR that logs the output of the test results to a file, however i see very little value in that, as we can get the full logs from the CI job today (without this PR, though you'll see a lot more in the logs files - stuff before the test task starts in the job).

**Comment by rchiodo** [comment]
> I have a simple PR that logs the output of the test results to a file, however i see very little value in that, as we can get the full logs from the CI job today (without this PR, though you'll see a lot more in the logs files - stuff before the test task starts in the job).

When we discussed this in standup, we felt like having 100s of MBS of logs in the output doesn't really help (you can't read them in the github site) and it was potentially a slowdown. 

So the thought was to just log to a file. No console logging whatsoever. Upload the logs as an artifact.

**Comment by rchiodo** [comment]
If the logs are downloadable and you can see which tests failed, seems done to me. 

We never checked that console logging slows anything down, but I doubt it does.
